.babelrc

```diff
@@ -1,4 +1,4 @@
 {
-  "presets": ["es2015", "react"],
```

Comment:
I'm hyper cautious of changes to babel, so I just wanted to make sure that this is all intentional and everything works just fine. Also, does this require any additional environment tweaks?

Reply:
Nope, this shouldn't require any other environment tweaks. Adding stage-2 to the babel-preset allows for use of the object spread operator in our code. The es2015 and react presets don't support it, but stage-2 does.